src/main/java/com/flywithingryd/IngrydAirways/model/Flight.java

```diff
@@ -39,6 +40,9 @@ public class Flight {
     @NotBlank
     @NotNull
     private double price;
+    @OneToMany
+    @JoinColumn(name = "")
+    private List<Itinerary> itineraryList;
```

Comment:
Since we wouldn't be needing the inverse code in itinerary, you might need to remove this

Reply:
U min the @OneToMany and List of itinerary?